Sources/App/Views/PackageController/PackageShow+View.swift

```diff
@@ -15,6 +15,10 @@ enum PackageShow {
             model.title
         }
 
+        override func bodyClass() -> String? {
+            "package"
+        }
```

Comment:
It would be cool if there was a CSS DSL such that this could be an actual type that's also referenced as a type in CSS :)